edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Core/routing/RoutingEdgeHub.cs

```diff
@@ -214,16 +216,28 @@ internal async Task ProcessSubscription(string id, Option<ICloudProxy> cloudProx
             }
         }
 
-        async void CloudConnectionEstablished(object sender, IIdentity identity)
```

Comment:
This is the main change? Instead of processing for a single identity, we are doing it for all identities.

Reply:
yes, that is right..